### PR TITLE
feat: use `linguist-languages`

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "standalone.js"
   ],
   "dependencies": {
+    "linguist-languages": "^6.3.0",
```
Comment (evilebottnawi) [inline package.json]: /cc @ikatyang should prettier export `linguist-languages` or each plugins should have own version? Using own version increase bundle size

Comment (ikatyang) [reply package.json]: `linguist-languages` is a dev dependency (though we placed it in `dependencies` to get `npm install prettier/prettier` work) that is supposed to be bundled (only the data for target languages will be bundled) so it _will_ increase the bundle size on the Prettier side if we're going to export it.

Comment (evilebottnawi) [reply package.json]: @ikatyang so, should we export this on prettier side or not? :smile:

Comment (ikatyang) [reply package.json]: Probably not, bundling all the language data (400+) with only a few (< 10) of them used does not sound like a good idea.

Comment (evilebottnawi) [reply package.json]: @ikatyang so we can merge this as is? Anyway we can use tree shaking to avoid unnecessary data in bundle

Comment (ikatyang) [reply package.json]: Sure.

(Sorry I missed the pending review, I thought you just cc me. 😅)

**Pull request merged**